02-FlexBox/assignment/css/_components.content.scss

```diff
@@ -1,13 +1,265 @@
+//remember REM SAVEREM and not pixels for layout
+
 html {
-    box-sizing: border-box;
+  box-sizing: border-box;
 }
 
 *,
 *:after,
 *:before {
-    box-sizing: inherit;
+  box-sizing: inherit;
```

Comment:
generally speaking a `* { box-sizing: border-box}` will suffice. Bringing in the inherit might be unneeded